src/docfx/build/document/Document.cs

```diff
@@ -320,6 +301,17 @@ internal static string PathToRelativeUrl(string path, ContentType contentType, s
             }
         }
 
+        private static ContentType GetContentType(Docset docset, FilePath path)
```

Comment:
confuse with above function with the same name `GetContentType`, maybe you can merge them into one function, or make the function name more specific

Reply:
yep, it could be combined to one